packages/create-cloudflare/templates/astro/templates/wrangler.toml

```diff
@@ -4,7 +4,9 @@ compatibility_date = "<TBD>"
 
 # Variable bindings. These are arbitrary, plaintext strings (similar to environment variables)
 # Note: Use secrets to store sensitive data.
-# Docs: https://developers.cloudflare.com/pages/functions/bindings/#environment-variables
+# Docs:
+# - https://developers.cloudflare.com/pages/functions/bindings/#environment-variables
+# - https://developers.cloudflare.com/pages/functions/bindings/#secrets
```

Comment:
Nit:
Docs for me is specific to `vars` here, so not to make things confusing for people I'd go with something like this instead:
```
# Note: Use secrets to store sensitive data (see: https://developers.cloudflare.com/pages/functions/bindings/#secrets).
# Docs: https://developers.cloudflare.com/pages/functions/bindings/#environment-variables
```

Reply:
(naturally this applies to all the other toml files as well 🙂)